lib/services.identity.js

```diff
@@ -376,6 +432,60 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
       404: 'The identity was not found on the system.'
     }
   });
+
+  app.delete(idPath, ensureAuthenticated, function(req, res, next) {
```

Comment:
You do need to have the IDENTITY_ADMIN permission to change identity status to deleted.